src/components/interactive/IconButton.md

```diff
@@ -0,0 +1,22 @@
+# Button
+
+Displays standard Buttons.
```

Comment:
It would be good to update this doc file:

```
# IconButton

Displays a button with an Icon and no label.

See <Button> for more details on event handling, spread props, and themes.
```